bitrise.yml

```diff
@@ -251,6 +251,61 @@ workflows:
         stack: osx-xcode-12.5.x
         machine_type_id: g2.8core
 
+  runL10nTests:
+    steps:
+    - activate-ssh-key@4:
+        run_if: '{{getenv "SSH_RSA_PRIVATE_KEY" | ne ""}}'
+    - git-clone@4: {}
+    - cache-pull@2: {}
+    - certificate-and-profile-installer@1: {}
+    - script@1:
+        inputs:
+        - content: |-
+            #!/usr/bin/env bash
+            # fail if any commands fails
+            set -e
+            # debug log
+            set -x
+
+            # Check if the build is a scheduled build
+
+            if [[ $BITRISE_GIT_MESSAGE == Schedule* ]]
+            then
+                echo "Scheduled build, running Full Functional Tests"
+                envman add --key TEST_PLAN_NAME --value FullFunctionalTests
+            else
+                echo "Regular build, running Smoke Test"
+                envman add --key TEST_PLAN_NAME --value SmokeTest
+            fi
+        - title: Check if build is scheduled or regular to set the test plan
+    - script@1:
+        inputs:
+        - content: |-
+            #!/usr/bin/env bash
+            ./checkout.sh
+        title: ContentBlockerGen
+    - script@1:
+        title: Set Default Web Browser Entitlement
+        inputs:
+        - content: |-
+            #!/usr/bin/env bash
+            set -x
+            /usr/libexec/PlistBuddy -c "Add :com.apple.developer.web-browser bool true" Focus.entitlements
+            /usr/libexec/PlistBuddy -c "Add :com.apple.developer.web-browser bool true" Klar.entitlements
+    - script@1:
+        title: Generate screenshots
+        inputs:
+        - content: |-
+            #!/usr/bin/env bash
+            set -x
+            ./screenshots.sh en-US
```

Comment:
This is just to try that the tests work for one locale, it will also be the workflow in charge of building the app and triggering the other builds that will only run (without building) the tests for each locale.
See similar [L10n](https://app.bitrise.io/app/6c06d3a40422d10f/workflow_editor#!/workflows?workflow_id=L10nBuild) workflow on firefox, and the [L10nScreenshotsTest](https://app.bitrise.io/app/6c06d3a40422d10f/workflow_editor#!/workflows?workflow_id=L10nScreenshotsTests)